src/set/set.service.ts

```diff
@@ -86,29 +86,35 @@ export class SetService {
     }
 
     async getSetsFromUser(
```

Comment:
With this solution the user can not see his private stuff when he is visiting his profile url when logged in. (I mean it's not necessary yet)

Reply:
Now he can hehe